src/ResultWriter/faultoutput.f90

```diff
@@ -388,6 +389,21 @@ SUBROUTINE calc_FaultOutput( DynRup_output, DISC, EQN, MESH, MaterialVal, BND, t
           ELSE
              P_f = 0.0
           ENDIF
+
+          if (EQN%FL.eq.33) then 
+             !case of ImposedSlipRateOnDRBoundary 'friction law': we add the additional stress to the fault output
+             !to show the imposed SR
+             Tnuc = DISC%DynRup%t_0
+             eta = (w_speed(2)*rho*w_speed_neig(2)*rho_neig) / (w_speed(2)*rho + w_speed_neig(2)*rho_neig)
+             IF (time.LE.Tnuc) THEN
```

Comment:
One should create a subroutine for Gnuc if used at several places.

Reply:
where would you put such a function within the code? 
a new file in ``SeisSol/src/Numerical_aux``? 
Apart from that, I think I fixed everything. Thx for the useful comments!

Reply:
You should put it where you would look for it first :) 
I'd put it in the Physics folder.

Reply:
Hi, 
Let me know how you like the function name I ve given. 
I will then also change the RS friction nucleation with the new function.

Reply:
Yes looks fine.